VideoLocker/res/values/strings.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">edX</string>
```

Comment:
To be clear, are you saying I need to add it even if we're going to override it? It seems to work as long as ``resValue`` is used with it.